Src/Base/AMReX_Geometry.H

```diff
@@ -62,7 +63,7 @@ public:
     RealBox prob_domain;
     Box domain;
     Real dx[AMREX_SPACEDIM];
-    int is_periodic[AMREX_SPACEDIM];
+    int is_periodic[AMREX_SPACEDIM]; /**< For each dimension, 0 if the domain is non-periodic and 1 if it is. */
```

Comment:
Does it make sense to add doxygen comments to the constructor and `define()` methods explaining this, instead of / in addition to commenting the member?

Reply:
Yes. I don't see why it couldn't be added to both. 

A positive of documenting the member this way is that you can search `is_periodic` and it will take you to the information. 

I added to the doxy comments on `define()` because it seemed natural to explain each variable there. What information would you want to put with the constructor? or should we just add to the class description?

Reply:
See my push for what I had in mind.

Reply:
Looks great! I also think this is a good general guideline. I'll add it to the Doxygen best practices guide.